src/Controller/WishListController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Product;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+
+class WishListController extends AbstractController
+
+{
+    /**
+     * @Route("/wishlist", name="wishlist")
+     */
+    public function index()
```

Comment:
zgodnie z dobrymi praktykami funkcje powinny się znaleźć w jednym miejscu w klasie. Standardowa kolejność to:
stałe
zmienne
funkcja __construct()
publiczne funkcje
funkcje protected
prywatne funkcje